core/src/brute/bosses.ts

```diff
@@ -59,6 +62,26 @@ const bosses: Boss[] = [
     disarm: panther.disarm,
     damage: panther.damage,
     reach: 3,
+    count: 1,
+  },
+  {
+    name: BossName.Cerberus,
+    base: PetName.dog1,
+    scale: 2.15,
+    initiative: 1.5,
+    strength: dog1.strength * 7.5,
+    agility: dog1.agility,
+    speed: dog1.speed,
+    hp: 30000,
+    counter: dog1.counter,
+    combo: 0,
+    block: dog1.block,
+    evasion: -0.2,
+    accuracy: 0.75,
+    disarm: dog1.disarm,
+    damage: dog1.damage,
+    reach: 1,
+    count: 3,
   },
```

Comment:
This boss is supposed to be easier than the other, here, if I understand correctly, there will be 3 dogs with 30k HP each, it seems too much for me